spec/requests/api/v0/rg_post_article_spec.rb

```diff
@@ -0,0 +1,45 @@
+RSpec.describe Api::V0::ArticlesController, type: :request do
+
+  describe 'research group POST article' do 
```

Comment:
the describe and it block descriptions in this file is not that descriptive.

e.g

`describe 'Research group can create article successfully'`
`it 'returns 200 response'`

Reply:
All the naming of the variables and assertions have been corrected.